openfl/federated/plan/plan.py

```diff
@@ -287,7 +310,7 @@ def get_assigner(self):
 
         return self.assigner_
 
-    def get_tasks(self):
+    def get_tasks(self) -> Dict[str, Dict[str, Any]]:
```

Comment:
Probably we need to create `Task` type to strictly specify the contained arguments and fields.